### DISCUSSION TITLE
Component Tokens breaking the link in the first usage

### DISCUSSION THREAD

**Discussion by TenreiroA** [six7/figma-tokens#2367] [Q&A]
Hi, everyone! 👋
I hope you are all great.
I'm having an issue with Tokens Studio that I've never added.
I've searched if there is any solution for this, but I couldn't find it...
I'm creating typography.
We have the core, the semantic and the components tokens.
When I insert the semantic token in the component, the first core component appears broken.
The second is normal. I don't understand why…
Has anyone had a similar problem and knows the solution to that?
Thank you so much 🌻

![image (3)](https://github.com/tokens-studio/figma-plugin/assets/139246606/306a4a26-8ea7-4cba-b152-6470925ccc5c)

**Comment by six7** [comment]
There's an issue around this currently where we're releasing a fix soon: https://github.com/tokens-studio/figma-plugin/issues/2301
